frontend/public/components/utils/webhooks.tsx

```diff
@@ -42,30 +59,49 @@ export const WebhookTriggers: React.SFC<WebhookTriggersProps> = ({ resource }) =
       : <span className="text-muted">No secret</span>;
   };
 
+  const copyWebhookToClipboard = trigger => {
+    const triggerProperty = getTriggerProperty(trigger);
+    const secretName = _.get(trigger, [triggerProperty, 'secretReference', 'name']);
+    k8sGet(SecretModel, secretName, namespace).then((secret: K8sResourceKind) => {
+      const webhookSecretValue = Base64.decode(secret.data.WebHookSecretKey);
```

Comment:
What happens if the secret does not actually have the key?

Reply:
Good question. One approach could be to return the obfuscated webhookURL as we did till now.
Other possibility is an error modal with a custom msg, something like, `Secret referenced in the ${webhookType} webhook trigger does not contain 'WebHookSecretKey' key.` .. or similar.
Thoughts ?

Reply:
I'd suggest we show an error modal. We should let the user know that the webhook won't work due to an invalid configuration.